src/app/view/perspectives/post-it/util/config-helper.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Store} from '@ngrx/store';
+import {Subscription} from 'rxjs/Subscription';
+import {AppState} from '../../../../core/store/app.state';
+import {PostItConfigModel} from '../../../../core/store/views/view.model';
+import {selectViewPostItConfig} from '../../../../core/store/views/views.state';
+
+export class ConfigHelper {
```

Comment:
What is the point of this class? It seems that it does not do anything special, just adds more code that would otherwise not be needed since a single line with a subscription could be placed directly in the component.